ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -104,12 +104,16 @@ - (void)stopObserving {
 
 - (DeviceType) getDeviceType
 {
+#if (!TARGET_OS_OSX)
     switch ([[UIDevice currentDevice] userInterfaceIdiom]) {
         case UIUserInterfaceIdiomPhone: return DeviceTypeHandset;
         case UIUserInterfaceIdiomPad: return DeviceTypeTablet;
         case UIUserInterfaceIdiomTV: return DeviceTypeTv;
         default: return DeviceTypeUnknown;
     }
+#endif
+    return DeviceTypeUnknown;
```

Comment:
if TARGET_OS_OSX DeviceTypeComputer ? Or DeviceTypeMac ? something like that would work for me

Reply:
Sounds good, I'll do DeviceTypeMac